go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.14.6
 	sigs.k8s.io/scheduler-plugins v0.25.7
 	sigs.k8s.io/yaml v1.3.0
-	volcano.sh/apis v1.2.0-k8s1.19.6
+	volcano.sh/apis v1.7.0
```

Comment:
Should we update the volcano version in a separate PR?

Reply:
e2e test  add in PR#1831